db/migrate/20240509194515_create_episode_feeds.rb

```diff
@@ -0,0 +1,41 @@
+class CreateEpisodeFeeds < ActiveRecord::Migration[7.0]
+  include TextSanitizer
+
+  def up
+    create_table :episodes_feeds, primary_key: [:episode_id, :feed_id] do |t|
+      t.belongs_to :episode
+      t.belongs_to :feed
+    end
```

Comment:
Looks like if I insert duplicate records, I get dupes in `Episode#feeds`, which seems like it could cause UI weirdness and other issues.. 

A unique constraint here seems appropriate to eliminate (and find causes) for inserting dupes.

Also, might be worth an index on these two columns -- probably a composite index (and avoid hitting the table at all).

Reply:
Hrm... it shouldn't be letting you insert dups, since it has a composite primary key.  Does that not do what I think it does?  (And it's also and index, since it is the primary key).

Reply:
Sorry, I had a stale branch checked out.  I see it now:

```
feeder_development=# \d+ episodes_feeds
                                       Table "public.episodes_feeds"
   Column   |  Type  | Collation | Nullable | Default | Storage | Compression | Stats target | Description 
------------+--------+-----------+----------+---------+---------+-------------+--------------+-------------
 episode_id | bigint |           | not null |         | plain   |             |              | 
 feed_id    | bigint |           | not null |         | plain   |             |              | 
Indexes:
    "episodes_feeds_pkey" PRIMARY KEY, btree (episode_id, feed_id)
    "index_episodes_feeds_on_episode_id" btree (episode_id)
    "index_episodes_feeds_on_feed_id" btree (feed_id)
Access method: heap
```

Reply:
Ah, my bad.  Yeah, i started with the rails built-in `has_and_belongs_to_many` table - but ended up switching.  Probably should have just added a 2nd migration.  They're free after all!